src/core/network.js

```diff
@@ -74,7 +74,7 @@ if (typeof PDFJSDev !== 'undefined' && PDFJSDev.test('FIREFOX || MOZCENTRAL')) {
       // blocked, e.g. via the connect-src CSP directive or the NoScript addon.
       // When this error occurs, this feature detection method will mistakenly
       // report that moz-chunked-arraybuffer is not supported in Firefox 37-.
-      x.open('GET', 'https://example.com');
+      x.open('GET', self.location.href);
```

Comment:
Please change this to `globalScope.location.href` instead, and add
```javascript


var globalScope = sharedUtil.globalScope;


```

at line https://github.com/mozilla/pdf.js/blob/master/src/core/network.js#L33 above.